File: src/main/java/my/maryvk/maryvkweb/service/RegisteredSeekerService.java
package my.maryvk.maryvkweb.service;

import my.maryvk.maryvkweb.domain.RegisteredSeeker;

import java.util.List;

public interface RegisteredSeekerService {
    void register(Integer targetId);
    void unregister(Integer targetId);
    List<RegisteredSeeker> findAll();
}

File: src/main/java/my/maryvk/maryvkweb/service/RegisteredSeekerService.kt
package my.maryvk.maryvkweb.service

import my.maryvk.maryvkweb.domain.RegisteredSeeker

interface RegisteredSeekerService {
    fun register(targetId: Int)
    fun unregister(targetId: Int)
    fun findAll(): List<RegisteredSeeker>
}